cmake/modules/RootConfiguration.cmake

```diff
@@ -676,10 +676,13 @@ get_filename_component(ROOT_BINDIR \"\${_thisdir}/${ROOT_CMAKE_TO_BIN_DIR}\" ABS
 # Deprecated value ROOT_BINARY_DIR
 set(ROOT_BINARY_DIR_SETUP "
 # Deprecated value, please don't use it and use ROOT_BINDIR instead.
-get_filename_component(ROOT_BINARY_DIR \"\${_thisdir}/${ROOT_CMAKE_TO_BIN_DIR}\" ABSOLUTE)
+get_filename_component(ROOT_BINARY_DIR \"\${ROOT_BINDIR}\" ABSOLUTE)
 ")
 set(ROOT_MODULE_PATH "\${_thisdir}/modules")
 
+message(STATUS ${ROOT_BINARY_DIR})
+message(STATUS ${ROOT_BINDIR})
```

Comment:
Not here. We want to see this from the point of view of an external project. But I see that in that case it should be fine in any case.